MinimalCirclesPath.py

```diff
@@ -1,4 +1,9 @@
 from math import sin, cos, radians, asin, atan2, sqrt, degrees, ceil, floor
+import matplotlib.pyplot as plt
+import geopandas as gpd
```

Comment:
should we add these to the requirements.txt? I know you dont need it for the actual app but we may still want it there

Reply:
sure we can do that